docs/ENV.md

```diff
@@ -70,6 +70,9 @@ AWS secret access key.
 #### `AWS_REGION`
 Region of the AWS IAM user.
 
+#### `AWS_STORAGE_BUCKET_NAME`
+AWS bucket where Django storages uploads the files
```

Comment:
seems like some typo here.

Reply:
Storages?
No. Django storages is the package name which handles the s3 storage

Reply:
then better to use `django-storages` or else it's weird to read.